src/init/init.ts

```diff
@@ -247,6 +253,41 @@ function createSelectPrompt(env: YeomanEnvironment): InquirerQuestion {
 }
 
 /**
+ * Checks and returns proxy settings from gitconfig. This function
+ * assumes that gitconfig will either be located locally (project's config file)
+ * or globally (user home)
+ */
+function getProxyConfig() {
+  //
+  let pathToConfig = './.git/config';
```

Comment:
The idea behind this was, that if user and proxy configured for git then he will need it for apis also.

So, should i put it only behind flag or do you still want me to take path also?